docs/README.md

```diff
@@ -1027,15 +1029,41 @@ where the schema for `CreateProductInput` is
 }
 ```
 
+#### Scheduling a command
+
+Scheduled commands are the way to add automated tasks to your application, like checking an e-commerce abandoned carts every two hours to send notifications to the customer to come back and complete the checkout. Booster scheduled commands are TypeScript classes decorated with `@ScheduledCommand`, and unlike conventional commands, their handle function doesn't have any parameters.
+
+In Booster, a scheduled command looks like this:
+
+```typescript
+@ScheduledCommand({
+  minute: '0/5', // runs every 5 minutes
+})
+export class CheckCartCount {
+  public static async handle(): Promise<void> {
+    /* YOUR CODE HERE */
+  }
+}
+```
+
+Notice that you can pass as parameters `minute`, `hour`, `day`, `month`, `weekDay` and `year` to set up a cron expression. By default, if no paramateres are passed, the scheduled command will not be triggered.
+
+#### Creating a scheduled command
+
+The preferred way to create a scheduled command is by using the generator, e.g.
+
+```shell
+boost new:scheduled-command CheckCartCount
```

Comment:
One nice thing we could do in this generator is giving the user a chance to define the cron schedule right in the CLI like this:

```shell
boost new:scheduled-command CheckCartCount --minutes "0/5"
```

Anyway, let's merge this as it is today and do this in a separate issue, this PR is long enough!